Index: back/app/src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Datasource (MySQL Config)\r\nspring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver\r\nspring.datasource.url=jdbc:mysql://localhost:3306/finalproject?useUnicode=true&characterEncoding=utf8&serverTimezone=UTC\r\nspring.datasource.username=root\r\n\r\nspring.datasource.password=qkqh1234!\r\n\r\n\r\n#spring.datasource.hikari.connection-init-sql=set names utf8mb4\r\nspring.datasource.tomcat.init-s-q-l= SET NAMES utf8mb4 COLLATE utf8mb4_unicode_ci\r\n\r\n# JPA\r\n\r\nspring.jpa.hibernate.ddl-auto=update\r\n\r\nspring.jpa.generate-ddl=true\r\nspring.jpa.show-sql=true\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/back/app/src/main/resources/application.properties b/back/app/src/main/resources/application.properties
--- a/back/app/src/main/resources/application.properties	(revision 8dae48f25ae440d74ecdd845dd539262f45372ed)
+++ b/back/app/src/main/resources/application.properties	(date 1663322542184)
@@ -3,7 +3,7 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/finalproject?useUnicode=true&characterEncoding=utf8&serverTimezone=UTC
 spring.datasource.username=root
 
-spring.datasource.password=qkqh1234!
+spring.datasource.password=Fleur0320!@#
 
 
 #spring.datasource.hikari.connection-init-sql=set names utf8mb4
@@ -11,7 +11,7 @@
 
 # JPA
 
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=create
 
 spring.jpa.generate-ddl=true
 spring.jpa.show-sql=true
